chrome/content/zotero/xpcom/zotero.js

```diff
@@ -507,6 +509,35 @@ Components.utils.import("resource://gre/modules/osfile.jsm");
 			}
 		}
 		
+		// Check for DB purge
+		var newAccountFile = dataDir.clone();
+		newAccountFile.append('new-account');
```

Comment:
This isn't for a new library — it's to "log out" of Zotero. We've wanted a "lab mode" like this for years, and it's a pretty big omission right now. People shouldn't need to know anything about the data directory to remove their data from a computer (e.g., a shared computer).